test/TestInternalGrains/PersistenceTestGrains.cs

```diff
@@ -668,9 +668,9 @@ private void CheckRuntimeEnvironment()
             var callStack = new StackTrace();
 
             executing = true;
-            Assert.AreEqual(_scheduler, scheduler, "Wrong TaskScheduler {0} Caller:{1}", scheduler, callStack);
-            Assert.IsNotNull(context, "Null ActivationContext -- Expected: {0} Caller:{1}", _context, callStack);
```

Comment:
Can these diag / extra info messages be preserved, so that someone gets more detailed context about break rather than having to go back through the code to work out what should have happened? 
Looks like you preserve the diag messages in most other places.